libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -38,37 +36,20 @@ export const AccountManager = ({
     },
     [gridRef]
   );
-  const { data, error, loading } = useDataProvider<AccountFields[], never>({
+  const { data } = useDataProvider<AccountFields[], never>({
     dataProvider: aggregatedAccountsDataProvider,
     update,
     variables,
   });
   dataRef.current = data;
-  const getRows = async ({
-    successCallback,
-    startRow,
-    endRow,
-  }: GetRowsParams) => {
-    const rowsThisBlock = dataRef.current
-      ? dataRef.current.slice(startRow, endRow)
-      : [];
-    const lastRow = dataRef.current?.length ?? -1;
-    successCallback(rowsThisBlock, lastRow);
-  };
   return (
-    <AsyncRenderer loading={loading} error={error} data={data}>
-      {data && (
-        <AccountTable
-          rowModelType={data?.length ? 'infinite' : 'clientSide'}
-          rowData={data?.length ? undefined : []}
-          ref={gridRef}
-          datasource={{ getRows }}
```

Comment:
I don't think I should use datasource in this scenario.  @bglownia 

![Screenshot 2022-10-10 at 13 14 17](https://user-images.githubusercontent.com/16125548/194875206-b101685f-c74a-44c9-9cf2-c320e1a3b4ae.png)

Reply:
It is not consistent with Deposits and Withdrawals. The Async renderer would simply display "No data" and no headers. I have the logic to display the headers and `No accounts` overlay within the account table. It is consistent with `Deposits` and `Withdrawals` and it looks quite clean. 

![image](https://user-images.githubusercontent.com/16125548/194898633-39d200c6-48ca-4efa-8dcb-d87bced8ff9a.png)
![Screenshot 2022-10-10 at 13 04 45](https://user-images.githubusercontent.com/16125548/194898688-fceb11f7-1a8b-44a2-8199-6b376426bf75.png)